connectionbuddy/build.gradle

```diff
@@ -4,12 +4,12 @@ apply from: '../config/quality.gradle'
 apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
 
     defaultConfig {
-        minSdkVersion 9
-        targetSdkVersion 25
+        minSdkVersion 19
```

Comment:
Why would we support 14? I think we, as developers, should push the usage of the latest firmware a bit more. I don't even like the fact that I've put it to 19. In my opinion, it should be 21, but I didn't want to be that strict since I know you are working on some projects with minSdkVersion 19.